packages/styles/src/utilities/_variables.scss

```diff
@@ -31,6 +31,144 @@ $position-values: (
 );
 
 $utilities: (
+  'border': (
+    responsive: true,
+    property: border,
+    class: border,
+    values: (
+      null: var(--post-border-width) var(--post-border-style)
+        rgba(var(--post-border-color-base), var(--post-border-opacity)),
+      0: 0,
+    ),
+  ),
+  'border-top': (
+    property: border-top,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-end': (
+    property: border-right,
+    class: border-end,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-bottom': (
+    property: border-bottom,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-start': (
+    property: border-left,
+    class: border-start,
+    values: (
+      null: var(--post-border-width) var(--post-border-style) var(--post-border-color),
+      0: 0,
+    ),
+  ),
+  'border-width': (
+    responsive: true,
+    css-var: true,
+    css-variable-name: border-width,
+    class: border,
+    values: from-tokens('border-widths', 'border'),
+  ),
+  'border-color': (
+    property: border-color,
+    class: border,
+    local-vars: (
+      'border-opacity': 1,
+    ),
+    values: from-tokens('border-colors', 'border'),
+  ),
+  'border-opacity': (
+    css-var: true,
+    class: border-opacity,
+    values: (
+      0: 0,
+      10: 0.1,
+      25: 0.25,
+      50: 0.5,
+      75: 0.75,
+      100: 1,
+    ),
+  ),
+  'rounded': (
+    property: border-radius,
+    class: rounded,
+    values: (
```

Comment:
This kind of border utilities should get their values from the tokens, using the `from-tokens` function.